### DISCUSSION TITLE
mqtt funktioniert / html nicht

### DISCUSSION THREAD

**Discussion by ThomasInOndrup** [john30/ebusd#956] [Q&A]
Zunächst herzlichen Dank für die tolle Arbeit, die zu ebusd und dem Adapter geführt haben! 

Mit folgenden Parametern habe ich ebusd auf meinem raspberry pi (192.168.179.106) konfiguriert:

EBUSD_OPTS="--scanconfig=full -d enh:192.168.179.22:9999 --mqttint=/etc/ebusd/mqtt-integration.cfg --mqtthost=127.0.0.1  --mqttport=1883  -p 8888 --httpport=8889"

Damit schickt ebusd zuverlässig mqtt-Nachrichten, aber über http://192.168.179.106:8889/data bekomme ich keine Ausgabe.

Hat jemand eine Idee, woran es hängen könnte?

**Comment by knalltuete86** [comment]
Existiert der Pfad `/var/ebusd/html` (Default value für --htmlpath)?

**Reply by ThomasInOndrup** [reply]
Ja, ich hatte mich zunächst auch gewundert, dass der Pfad nicht existiert. Aber auch die nachträgliche Anlage des Verzeichnisses hat nicht geholfen.
